### DISCUSSION TITLE
2023-08-11

### DISCUSSION THREAD

**Discussion by yeopto** [woowacourse-teams/2023-haru-study#305] [daily_meeting]
### 오늘의 할 일
- 마코,테오 - OAuth 로그인 기능 마무리
- 히이로, 모디 - 테스트 서버
  - main 브랜치를 3차 데모데이 이전으로 되돌리기
  - 현재 mysql DB 서버 OAuth 붙이기 이전으로 롤백한 뒤 스냅샷 저장해두기 (운영 DB)
  - mysql 테스트 서버용 DB 생성하기
  - production ec2에 테스트 서버 올리기
  - 기존 젠킨스 배포 파이프라인 코드를 테스트 서버 파이프라인으로 구축하기
  - 젠킨스에 프로덕션 서버용 파이프라인 새로 구축하기
  - 젠킨스 파이프라인 코드도 어떻게 관리할 지 생각해봐야 함
    - 프로덕션과 테스트 배포 코드가 함께 git 에서 관리되는 젠킨스 배포 스크립트를 기준으로 돌아가게 되기 때문이다.
- 프론트 - Modal, 프론트엔드 요구사항(webpack), 이슈 분배

### 언젠가 할 일
- 소켓, 폴링 양방향 통신 어떻게 할지 (4차 스프린트 내에 그냥 진행하고 발표만 안하는 방향으로 갈건지 로그인 이후 결정하기)
- 질문 PlaceHolder로 유의미한 응답 예시를 제공하는 기능 (목표설정 가이드라인)
- 브랜치 전략 결정하기(dev -> main)
- A라는 스터디를 참여후 B스터디 참여한 뒤 다시 A스터디에 참여할 시 닉네임을 찾지 못한 이슈(로그인이 된다면 해결될 문제이긴 함)

### 개인 일정
- 노아 5시 근로

**Comment by yeopto** [comment]
### 체크인

5점 - 발표 준비까지 할게 많네요..

### 체크아웃

8점 - 연휴가 되어버려서 기분이 좋습니다.

**Comment by nlom0218** [comment]
### 체크인

9점 - 비가와서 날씨가 선선하지만 슬리퍼 신으니 양말이 꺼림칙하네요

**Comment by woo-jk** [comment]
### 체크인
9점 - 잠을 너무 늦게자서 -2 금요일이라 +1

**Comment by woosung1223** [comment]
### 체크인
10점 - 카드가 안보여서 슬프지만 어제 치킨 먹어서 좋네요

**Comment by jaehee329** [comment]
### 체크인
6점 - 왜 주말이 아니죠?

**Comment by aak2075** [comment]
### 체크인

10점 - 즐거운 금요일이네요 마지막까지 화이팅합시다💪

**Comment by MoonJeWoong** [comment]
### 체크인
9점 - 어제 백엔드 팀에서 미뤄뒀던 PR을 검토하고 정리했는데 기분좋게 close할 수 있을 것 같습니다. 한 주 마무리 힘내서 달려보시죠!
